cli/tests/integration/coverage_tests.rs

```diff
@@ -51,36 +52,37 @@ fn error_if_invalid_cache() {
   std::fs::copy(mod_test_path, mod_test_temp_path).unwrap();
 
   // Generate coverage
-  let status = util::deno_cmd_with_deno_dir(&deno_dir)
-    .current_dir(deno_dir_path)
-    .arg("test")
-    .arg("--quiet")
-    .arg(format!("--coverage={}", tempdir.to_str().unwrap()))
-    .stdout(std::process::Stdio::piped())
-    .stderr(std::process::Stdio::inherit())
-    .status()
-    .unwrap();
+  let context = TestContext::default();
+  let deno_dir_str = deno_dir_path.to_str().unwrap();
+  let output = context
+    .new_command()
+    .env("DENO_DIR", deno_dir_str)
```

Comment:
If `let context = TestContext::default()` were changed to `let context = TestContext::new().use_temp_cwd().build()`, then moved to the top and the deno dir was retreived via `let deno_dir = context.deno_dir();`, then this line and the below could be removed I think.

Reply:
It's working (with `TestContextBuilder::new`)